psrm_ci_ft_base.py

```diff
@@ -115,8 +117,13 @@ def check_id(self, id):
     from default_paths import path_v4, v4
     from psrm_utils import cache_psrm
 
+    # load psrm data with cache
+    psrm = cache_psrm(path=path_v4, input=v4)
+
+    # make a nicely formatted excel sheet
+    utils.df_to_excel(psrm.afregning.head(50))

```

Comment:
```suggestion
    utils.df_to_excel(psrm.afregning.sample(50))
```
Bare et forslag, så får man 50 tilfældige hver gang :)